moncli/entities/column_value/simple.py

```diff
@@ -49,6 +51,28 @@ def __isint(self, value):
 class PeopleValue(ComplexNullValue):
     """A people column value."""
 
+    native_type = list
+    native_default = []
+    allow_casts = ()
+
+    def _convert(self, value):
+        value_list = value['personsAndTeams']
+        return [PersonOrTeam(**value_data) for value_data in value_list]
+    
+    def _format(self):
+        personsAndTeams = []
+        for list_item in self.value:
+            if not isinstance(list_item,PersonOrTeam):
+                raise ColumnValueError(
+                    'invalid_people_value',
+                    self.id,
+                    'Invalid person or team value "{}".'.format(list_item)
+                )
+            id = list_item.id
+            kind = list_item.kind
+            personsAndTeams.append({'id': id,'kind':kind})
```

Comment:
When attempting to add a new person to a people column, this throws an error.  The 'kind' assignment needs to be kind.name.  The way you have it you are passing the entire enum back.

Please test your code with the API before submitting it to me.